bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/asset/FileAssetIngestor.java

```diff
@@ -57,6 +59,8 @@
  */
 public class FileAssetIngestor extends AssetIngestor {
 
+    static String SFTP_URL_ENCODING = "utf-8";
```

Comment:
Can we move this into a more general location, e.g. since the HierarchialElement class is referencing this, maybe that's a more suitable place for this constant?

Reply:
Sure, bu code climate has forced me to do so : cba38410d96b66ffba5d304dc8b8a62dd1c5e86e because of constant in Interface. I will move it into separate class as well as encodeUriParts method in HierarchialElement. It require additional work for urls with port

Reply:
Yes please.  Thanks so much!  That picky code climate!!